devicemap-schema.json

```diff
@@ -36,6 +36,7 @@
               "HASync",
               "HAFabric",
               "MGMT",
+              "SWITCH",
```

Comment:
`SWITCHED` or `SWITCHED_PARENT`? This makes it sound like it is just connected to a switch